doc/getting_started/toolchain_3rd_party_x_compilers.rst

```diff
@@ -65,24 +70,29 @@ Intel ISSM
       Like the Zephyr repository, do not install the toolchain in a directory
       with spaces anywhere in the path.
 
-#. Configure the environment variables needed to inform the Zephyr build system
-   to use this toolchain:
+#. :ref:`Set these environment variables <env_vars>`:
 
    - Set :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to ``issm``.
    - Set :envvar:`ISSM_INSTALLATION_PATH` to the directory containing the
      extracted files.
 
-   For example:
+#. To check that you have set these variables correctly in your current
+   environment, follow these example shell sessions (the
+   :envvar:`ISSM_INSTALLATION_PATH` values may be different on your system):
```

Comment:
This introduction is too verbose, even more so now that there's an entire page about environment variables. It can be replaced with a single word: "Examples:"

Reply:
I think a plain "examples" would easily be misconstrued by the reader to mean "examples of how to set these variables" (which they are not ), rather than "examples of how to check you have set these variables correctly" (which they are).

Setting up a toolchain is the first time zephyr users are asked to set environment variables. I have watched users stumble over this process again and again, both in person and on IRC. It's probably one of the most common support issues for beginners, so as a special case, I am trying to make it extra clear how they can check their work.

Just FYI, this PR is also a stopgap and prep work for moving the getting started toolchain configuration to west. The content in this section will eventually be replaced with something like this:

```
west config build.zephyr_toolchain_variant issm
west config build.issm_installation_path /home/you/Downloads/issm0-toolchain-windows-2017-02-07
```

I am just trying to help avoid support requests in the meantime.

If you feel strongly that this is too verbose I'll just delete these "To check..." entirely, but I think that's a bad idea for reasons explained above.

Reply:
Perhaps it's time for a short usability test with some new Zephyr users to verify these new getting started documents are doing their job.

Reply:
I think that's a great idea, and one that perhaps @Thea-Aldrich could follow-up on?

Reply:
That's a great idea @carlescufi. I have a couple folks I can reach out to in the community who are interested in providing feedback. @dbkinder would you like to team up on drafting the questions and super informal methodology ya'll would like me to use as I gather feedback?

Reply:
> Perhaps it's time for a short usability test with some new Zephyr users to verify these new getting started documents are doing their job.

@dbkinder I agree, but wonder if that should block the merge or if we can get a more explicit verdict from you on the point being discussed here?

Reply:
Gosh no - don't block this PR merge for a usability test (we can update as needed later).

Reply:
BTW, there actually is a usability study going on. Let's get this PR merged and we'll address issues found during this study in a subsequent PR.